finder-app/manual-linux.sh

```diff
@@ -34,47 +42,89 @@ if [ ! -e ${OUTDIR}/linux-stable/arch/${ARCH}/boot/Image ]; then
     echo "Checking out version ${KERNEL_VERSION}"
     git checkout ${KERNEL_VERSION}
 
-    # TODO: Add your kernel build steps here
+    # Kernel build steps
+    make ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} mrproper    # Deep clean prev configs
+    make ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} defconfig   # Configure virt arm dev board
+    make -j4 ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} all     # Build vmlinux target, kernel image
+    make ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} modules     # Build kernel modules
+    make ARCH=${ARCH} CROSS_COMPILE=${CROSS_COMPILE} dtbs        # Build the device tree
+
 fi
 
 echo "Adding the Image in outdir"
+cp ${OUTDIR}/linux-stable/arch/${ARCH}/boot/Image ${OUTDIR}
 
 echo "Creating the staging directory for the root filesystem"
 cd "$OUTDIR"
 if [ -d "${OUTDIR}/rootfs" ]
 then
 	echo "Deleting rootfs directory at ${OUTDIR}/rootfs and starting over"
     sudo rm  -rf ${OUTDIR}/rootfs
+    mkdir ${OUTDIR}/rootfs
+else
+    mkdir ${OUTDIR}/rootfs
```

Comment:
@betai97 You could move the mkdir line out of the if check to avoid duplicating in both branches.